packages/design-system/src/components/List/List.stories.jsx

```diff
@@ -20,17 +20,51 @@ const listMarkup = (type) => {
 
   return (
     <>
-      <h2 className="ds-h6" id={`${type}-list-id`}>
-        {label} list title
-      </h2>
       <Tag className={className} aria-labelledby={`${type}-list-id`}>
-        <li>List item 1</li>
-        <li>List item 2</li>
+        <li>{type} list item 1</li>
+        <li>{type} list item 2</li>
```

Comment:
Good idea 👍

Reply:
Looks like the `label` variable is now unused and that that could work here
```suggestion
        <li>{label} list item 1</li>
        <li>{label} list item 2</li>
```